bundles/org.openhab.binding.modbus.helioseasycontrols/src/main/java/org/openhab/binding/modbus/helioseasycontrols/internal/HeliosEasyControlsHandler.java

```diff
@@ -623,6 +631,75 @@ public String formatDate(String variableName, ZonedDateTime date) {
         }
     }
 
+    private ArrayList<String> getMessages(long bitMask, int bits, String prefix) {
```

Comment:
It's good practice to use the base type where possible. Please check all.
```suggestion
    private List<String> getMessages(long bitMask, int bits, String prefix) {
```